.github/workflows/all-post-commit-workflows.yaml

```diff
@@ -19,6 +20,11 @@ jobs:
   build-artifact:
     uses: ./.github/workflows/build-artifact.yaml
     secrets: inherit
+  build-docker-image-2204:
```

Comment:
why is there a job only for `22.04`?

Reply:
to answer this Q, build-artifact.yaml (at the time this was reviewed) internally built the docker-image for 20.04.

Newer changes to PR have pulled out the build docker image step into its own job.